src/insns/acperm_32bit.adoc

```diff
@@ -52,9 +52,9 @@ The rules from <<acperm_rules>> must be followed when removing permissions.
 |  4  (RV32 only) | <<x_perm>>   | <<r_perm>>
 |  5  (RV32 only) | <<w_perm>>   | not(<<c_perm>>) or <<lm_perm>>
 |  6  (RV32 only) | <<x_perm>>   | <<w_perm>> or <<c_perm>>
-|  7              | <<el_perm>>  | <<c_perm>> and <<r_perm>>
-|  8              | <<lm_perm>>  | <<c_perm>> and <<r_perm>>
-|  9              | <<sl_perm>>  | <<c_perm>>
+|  7              | <<el_perm>>  | <<c_perm>> and <<r_perm>> and <<lm_perm>>
```

Comment:
This is basically `(quandrant == 3) || (quadrant == 1 && C)` which I guess lines up with these constraints. It's a bit weird to list LM as requirement for the unrelated EL permission, so this should be RV32 only.
```suggestion
|  7  (RV32 only) | <<el_perm>>  | <<c_perm>> and <<r_perm>> and <<lm_perm>>
```

Reply:
same comment as above